### PR TITLE
Update Travis CI yaml and add script

### PR DESCRIPTION
**Pull request by dwalintukan**
Updated the Travis CI to use `truffle test` and to use my script to create high value ether accounts for `TestRPC`.

I want to get this merged in before my `more_tests` branch. I am having problems getting Travis CI to do a new build test for my branch. So I think updating the `.travis.yaml` in a different branch first might solve this.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,11 @@ before_install:
...
+  - npm install -g ethereumjs-testrpc
+
+before_script:
+  - testrpc_high_value.sh
 
 script:
-  - npm test
\ No newline at end of file
+  - truffle test
+
```
Comment (frankobe) [inline .travis.yml]: nit. a space

File: testrpc_high_value.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+echo "Running TestRPC with high value accounts."
+
+# Running this script will create 4 high value accounts to use on TestRPC.
+# It will create 500,000 Ether for each account.
+
+# 1 public:	0xc8F3ea1bb2ad0B7fFb0955064A62a0556Ba8bea2
```
Comment (frankobe) [inline testrpc_high_value.sh]: remove the public/private comments?

File: .travis.yml (hunk 1 above)
Comment (dwalintukan) [reply .travis.yml]: fixed

File: testrpc_high_value.sh (hunk 2 above)
Comment (dwalintukan) [reply testrpc_high_value.sh]: removed

**Pull request merged**